### ISSUE TITLE
NotAuthenticated when syncing calendars

### ISSUE THREAD

**Issue by Jonny007-MKD** [jobisoft/TbSync#341]
## Your environment

TbSync version: 2.8
Thunderbird version: 68.2.2

[X] Yes, I have installed the latest available (beta) version from 
https://github.com/jobisoft/TbSync/releases 
and my issue is not yet fixed, I can still reproduce it.

## Expected behavior
Syncing works

## Actual behavior
Synchronization of address books works, but of calendars fails.

      ERROR: 401 (URL: https://.../remote.php/dav/calendars/myuser/calendar/ (PROPFIND)
      Request:
      <d:propfind xmlns:d="DAV:">
        <d:prop>
          <d:resourcetype />
          <d:displayname />
        </d:prop>
      </d:propfind>
      Response:
      <?xml version="1.0" encoding="utf-8"?>
        <d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
        <s:exception>Sabre\DAV\Exception\NotAuthenticated</s:exception>
        <s:message>
          No public access to this resource.,
          No 'Autorization: Bearer' header found. Either the client didn't send one, or the server is mis-configured,
          No 'Authoriztation: Basic' header found. Either the client didn't send one, or the server is mis-configured
        </s:message>
      </d:error>

## Steps to reproduce
I only added my account and started syncing

**Comment by jobisoft** [comment]
What kind of server is that? Can I have a test account on that server to be able to reproduce this? If so, please send credentials via email...

**Comment by jobisoft** [comment]
During account creation, you must specify a username, otherwise TbSync will switch to public / anonymous access. You can add a username to your TbSync account by disabling it (the account, not TbSync) and edit the connection details.

**Comment by Jonny007-MKD** [comment]
It's a NextCloud instance, and I specified the user name. My addressbooks are private as well :). You're going to receive a mail with your credentials in a few minutes. Thanks!

**Comment by jobisoft** [comment]
I have not received an email yet, just to make sure that you did nt´ot send it to the wrong adress. My email is listed on my main github page:

https://github.com/jobisoft/

**Comment by Jonny007-MKD** [comment]
Sorry, now it's on the way :)

**Comment by jobisoft** [comment]
works for me:
![grafik](https://user-images.githubusercontent.com/5830621/69991586-4359ab00-1548-11ea-95fc-560db6bf6fcf.png)

**Comment by jobisoft** [comment]
I just used automatic setup with this url:
https://<your own subdomain>.<something with north lights>.de

**Comment by Jonny007-MKD** [comment]
Thanks John! Then it may be some setting on my acocunt in NextCloud? I am going to look into it tomorrow.

**Comment by Jonny007-MKD** [comment]
Together with @jobisoft I was able to fix this issue.

I had migrated my Thunderbird profile from Windows to Linux. Due to a Windows path in the file pcks11.txt in the profile, the master password stuff did not work (I was unable to set a master password). This caused this error in the debug log of TbSync

    [Error adding loginInfo] : [Exception... "User canceled master password entry"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "JS frame :: resource://gre/modules/crypto-SDR.js :: encrypt :: line 90"  data: no]

After renaming the pcks11.txt in my profile to pcks11.txt.backup it is working now.

https://forums.linuxmint.com/viewtopic.php?t=293921

Thanks, @jobisoft!
